test/arrays/ferguson/semantic-examples/0-init-alias.chpl

```diff
@@ -0,0 +1,10 @@
+use samples;
+// var one = new R(1);
```

Comment:
These comments are just there to limit the amount of code one has to look at if debugging the test. I'd like to keep them, but they are just a quick-reference for any person looking closely at the test.